src/style.css

```diff
@@ -1,3 +1,78 @@
+:root {
+  --color-primary: #00acdc;
+}
+
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__logo {
+  margin-left: 50px;
+  margin-top: 10px;
+  width: 40px;
+  height: 40px;
+}
+
+.menu {
+  background-color: #fff;
+  margin-right: 50px;
+}
+
+.menu .menu__list {
+  list-style: none;
+  display: flex;
+  justify-items: left;
+  padding: 0;
+}
+
+.menu .menu__list .menu__item {
+  text-transform: uppercase;
+}
+
+.menu .menu__list .menu__item .menu__link {
+  text-decoration: none;
+  text-align: center;
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.menu .menu__list .menu__item .menu__link:not(.is-active) {
+  color: #000;
+}
+
+.menu .menu__list .menu__item .menu__link:not(.link-first) {
+  margin-left: 20px;
+}
+
+.menu .menu__list .menu__item .is-active {
+  color: var(--color-primary);
+  position: relative;
+}
+
+.menu .menu__list .menu__item .is-active::after {
+  content: '';
+  position: absolute;
+  color: var(--color-primary);
+  width: 90%;
+  bottom: 0;
+  border: 2px solid var(--color-primary);
+  border-radius: 10px;
+}
+
+.menu .menu__list .menu__item .menu__link:hover {
+  color: var(--color-primary);
 }
```

Comment:
Define each element as a single CSS rule, applying only the necessary styles to each one!
Every element should be unique and not depend on its position on the page (that is why the naming convention follows the block__element--modifier rule)

Using
```css
.menu .menu__list {
/*...styles*/
} 
```
Is not only unnecessary but discouraged! It creates a highly coupled and needlessly convoluted styles;

Use:
```css
.menu {
/*...menu styles*/
} 

.menu__list {
/* ...menu__list styles */
}
```